translationCoreApps/ScripturePane
# Feature - #2470 - Gray out (disable) sources that have already been loaded in Scripture Pane

#### This pull request addresses:

Addresses issue https://github.com/unfoldingWord-dev/translationCore/issues/2470

#### How to test this pull request:

Go to WA or tW. If you already have 3 sources selected, remove one. Then add a source to the Scripture pane. Any sources you still have selected should be grayed out and unselectable in the source select drop down. Test by removing all your sources, etc.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/translationcoreapps/scripturepane/75)
